snippets/react-native/package.json

```
@@ -5,15 +5,24 @@
     "main": "index.ts",
     "license": "MIT",
     "scripts": {
-        "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+        "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
+        "format": "eslint . --ext .js,.jsx,.ts,.tsx --fix"
```

Add compile for local testing

```suggestion
        "format": "eslint . --ext .js,.jsx,.ts,.tsx --fix",
        "compile": "tsc --noEmit"
```